independent-publisher-connectors/MaintenanceRequestOxmaint/apiDefinition.swagger.json

```diff
@@ -0,0 +1,152 @@
+            "required": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "title": {
+                  "type": "string",
+                  "description": "title",
+                  "title": "Title",
+                  "x-ms-visibility": "important"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description",
+                  "title": "Description",
+                  "x-ms-visibility": "important"
+                },
+                "priority": {
+                  "type": "string",
+                  "description": "priority",
+                  "title": "Priority",
+                  "x-ms-visibility": "important"
+                },
+                "requested_by": {
+                  "type": "string",
+                  "description": "requested_by",
+                  "title": "Requestor",
+                  "x-ms-visibility": "important"
+                },
+                "date": {
+                  "type": "string",
+                  "description": "date",
+                  "title": "Date",
+                  "x-ms-visibility": "important"
+                },
+                "master_email": {
+                  "type": "string",
+                  "description": "master_email",
+                  "title": "Master Email",
+                  "x-ms-visibility": "important"
+                },
+                "api_key": {
+                  "type": "string",
+                  "description": "api_key",
+                  "title": "API Key",
+                  "x-ms-visibility": "important"
+                }
+              },
+              "default": {
+                "title": "Test WO Request",
+                "description": "Hello there Please create a WO request",
+                "priority": "Low",
+                "requested_by": "Sachin Upadhyay",
+                "date": "20-Feb-2024 11:12 AM",
+                "master_email": "nikita@gmail.com",
+                "api_key": "123456"
+              },
+              "x-ms-visibility": "important",
+              "required": [
+                "api_key",
+                "date",
+                "description",
+                "master_email",
+                "priority",
+                "requested_by",
+                "title"
+              ]
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [],
+  "x-ms-openai-manifest": {
+    "humanName": ""
+  }

```

Comment:
Kindly refer the below link and add "x-ms-connector-metadata" property

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json#L25